modules/lead/product-base/main.tf

```diff
@@ -21,3 +21,21 @@ provider "kubernetes" {
 provider "helm" {
   alias = "system"
 }
+
+data "kubernetes_secret" "vault_root_token_secret" {
+  provider = kubernetes.staging
+  metadata {
+    namespace = "toolchain"
+    name      = "vault-root-token"
```

Comment:
maybe we should pass the `vault_root_token_secret` and `vault_namespace` as variables to the product so we don't have to hardcode these.  we can also use `vault_namespace` on line 34 as well

Reply:
Updated products to include variables `vault_root_token_name` and `vault_namespace` from `vault` module outputs.